src/components/form/range/_range_highlight.scss

```diff
@@ -9,11 +9,13 @@
  * GitHub history for details.
  */
 
+@use 'sass:math';
+
 .ouiRangeHighlight {
   position: absolute;
   left: 0;
   width: 100%;
-  top: calc(50% - #{($ouiRangeHighlightHeight / 2)});
+  top: calc(50% - #{math.div($ouiRangeHighlightHeight, 2)});
```

Comment:
This is one of the cases where `sass-lint` fails to parse this syntax, for reference